bdd100k/eval/seg_test.py

```diff
@@ -0,0 +1,44 @@
+"""Test cases for mot.py."""
+import os
+import unittest
+
+import numpy as np
+
+from .seg import evaluate_segmentation, fast_hist
+
+
+class TestIoU(unittest.TestCase):
+    """Test cases for the segmentation IoU computation."""
+
+    def test_fast_hist(self) -> None:
```

Comment:
It is more concise to use pytest syntax here

Reply:
Can I see some examples? I'm not familiar with pytest syntax

Reply:
Here are some https://github.com/scalabel/scalabel/blob/master/scalabel/label/io_test.py 🙂